File: C#_Base_PT4/XString14.cs
// File: "XString14"
using PT4;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace PT4Tasks
{
    public class MyTask : PT
    {
        public static async void Solve()
        {
            Task("XString14");

            string s = GetString();

            int n = s.Length;

            int cnt = 0;

            for(int i = 0; i < n; i++)
            {
                if (s[i] >= 'A' && s[i] <= 'Z')
                    cnt++;
            }
                
            Put(cnt);
        }
    }
}

File: C++_Base_PT4/XString14.cpp
#include "pt4.h"
using namespace std;
//给定一个字符串。 计算它包含的大写拉丁字母的数量
void Solve()
{
    Task("XString14");
	
	string str;
	int count=0;
	
	pt >> str;
	for(int i = 0; i < str.size(); i++)
	{
		if(str[i]>='A'&&str[i]<='Z')
		{
			count++;
		}
	}
	
	pt << count;
}
